### PR TITLE
osd: enhance logging for osd network error

### PR DESCRIPTION
**Pull request by Liuchang0812**
Signed-off-by: liuchang0812 <liuchang0812@gmail.com>

### CODE REVIEW

**Comment by Liuchang0812** [comment]
@tchaikov please take a look, It's useful when I do some network crash test to osd.

File: src/osd/OSD.cc [hunk 1]
```diff
@@ -7134,21 +7134,21 @@ void OSD::_committed_osd_maps(epoch_t first, epoch_t last, MOSDMap *m)
 	if (r != 0) {
 	  do_shutdown = true;  // FIXME: do_restart?
           network_error = true;
-          dout(0) << __func__ << " marked down: rebind failed" << dendl;
+          dout(0) << __func__ << " marked down: rebind cluster_messager failed" << dendl;
```
Comment (tchaikov) [inline src/osd/OSD.cc]: need to wrap line at 80 chars.

**Comment by tchaikov** [comment]
lgtm, just need to wrap the line.

**Comment by Liuchang0812** [comment]
@tchaikov it's done. thanks for your suggestion

File: src/osd/OSD.cc [hunk 2]
```diff
@@ -7134,21 +7134,24 @@ void OSD::_committed_osd_maps(epoch_t first, epoch_t last, MOSDMap *m)
 	if (r != 0) {
 	  do_shutdown = true;  // FIXME: do_restart?
           network_error = true;
-          dout(0) << __func__ << " marked down: rebind failed" << dendl;
+          dout(0) << __func__ 
+              << " marked down: rebind cluster_messager failed" << dendl;
```
Comment (hjwsm1989) [inline src/osd/OSD.cc]: there is a typo, should be cluster_messenger

Comment (Liuchang0812) [inline src/osd/OSD.cc]: thanks, my bad. I have fixed this typo.

**Comment by Liuchang0812** [comment]
@tchaikov @hjwsm1989 it's done.

**Comment by Liuchang0812** [comment]
@tchaikov hi, tchaikov, need your approval :smiley_cat:

**Comment by xiexingguo** [comment]
retest this please

**Comment by xiexingguo** [comment]
On behalf of @tchaikov (since he is on vacation).

**Comment by Liuchang0812** [comment]
@xiexingguo thanks :+1:

File: src/osd/OSD.cc [hunk 3]
```diff
@@ -7134,21 +7134,24 @@ void OSD::_committed_osd_maps(epoch_t first, epoch_t last, MOSDMap *m)
 	if (r != 0) {
 	  do_shutdown = true;  // FIXME: do_restart?
           network_error = true;
-          dout(0) << __func__ << " marked down: rebind failed" << dendl;
+          dout(0) << __func__
```
Comment (xiexingguo) [inline src/osd/OSD.cc]: It looks better if you do all the changes the same way and seems the preferred option is to be `<<` aligned here:
```


dout(0) << __func__ << " marked down:"
        << " marked down: rebind cluster_messenger failed" << dendl;
```

Comment (Liuchang0812) [inline src/osd/OSD.cc]: yes, it's better. I have updated this code. thanks :smile:

**Pull request merged**